features/logging/deploy_logging_operators_via_console.feature

```diff
@@ -0,0 +1,65 @@
+    Given the logging operators are redeployed after scenario
+    Given logging channel name is stored in the :logging_channel clipboard
+    Given I run the :create admin command with:
+      | f | https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/logging/clusterlogging/deploy_clo_via_olm/01_clo_ns.yaml |
+    Then the step should succeed
+    Given I register clean-up steps:
+    """
+    Given logging service is removed successfully
+    Then the step should succeed
+    """
+    Given "cluster-logging" packagemanifest's operator source name is stored in the :clo_opsrc clipboard
+    Given I switch to the first user
+    Given the first user is cluster-admin
+    Given I open admin console in a browser
+    # subscribe cluster-logging-operator
+    When I perform the :goto_operator_subscription_page web action with:
+      | package_name     | cluster-logging     |
+      | catalog_name     | <%= cb.clo_opsrc %> |
+      | target_namespace | openshift-logging   |
+    Then the step should succeed
+    And I perform the :set_custom_channel_and_subscribe web action with:
+      | update_channel    | <%= cb.logging_channel %> |
+      | install_mode      | OwnNamespace              |
+      | approval_strategy | Automatic                 |
+    Given cluster logging operator is ready
+
+  # @auther qitang@redhat.com
+  # @case_id OCP-24292
+  @admin
+  @destructive
+  Scenario: Deploy elasticsearch-operator via Web Console
+    # clear the logging related resources
+    Given logging service is removed successfully
+    Given the logging operators are redeployed after scenario
+    Given logging channel name is stored in the :logging_channel clipboard
+    Given "elasticsearch-operator" packagemanifest's operator source name is stored in the :eo_opsrc clipboard
+    Given I switch to the first user
+    Given the first user is cluster-admin
+    Given admin ensures "elasticsearch-operator" subscriptions is deleted from the "openshift-operators" project after scenario
+    Given I open admin console in a browser
+    When I perform the :goto_operator_subscription_page web action with:
+      | package_name     | elasticsearch-operator |
+      | catalog_name     | <%= cb.eo_opsrc %>     |
+      | target_namespace | openshift-operators    |
+    Then the step should succeed
+    When I perform the :set_custom_channel_and_subscribe web action with:
+      | update_channel    | <%= cb.logging_channel %> |
+      | install_mode      | AllNamespace              |
+      | approval_strategy | Automatic                 |
+    Then the step should succeed
+
+    Given I use the "openshift-operators" project
+    Given I wait for the "elasticsearch-operator" subscriptions to appear
+    And evaluation of `subscription("elasticsearch-operator").current_csv` is stored in the :eo_csv clipboard
+    Given admin ensures "<%= cb.eo_csv %>" clusterserviceversions is deleted from the "openshift-operators" project after scenario
```

Comment:
shouldn't this be `cluster_service_version`?

And should we register this clean-up step immediately before/after object was created?

Reply:
it should be `clusterserviceversions`  https://github.com/openshift/verification-tests/blob/master/lib/openshift/cluster_service_version.rb#L5

I have to get the name of the csv object before I delete it. If the subscription is not created successfully, the csv object would not be created.

Reply:
I believe `cluster_service_version` should work. These are registered as aliases - the readable name and the resource API name.

I'll update and merge. Let me know if you hit troubles with that. Thank you.

Reply:
OK, thank you very much.